test/testModel.cc

```diff
@@ -600,9 +612,9 @@ SUITE(Canvas)
         CHECK_EQUAL(2,model->numItems());
         CHECK_EQUAL(0,g->inVariables.size());
 
-        // move b into group
-        mouseDown(b->x(),b->y());
-        mouseUp(g->x(),g->y());
+        // move b into group. 
+        mouseDown(b->x()+5,b->y()+5);   
+        mouseUp(g->x()+5,g->y()+5);  // small offset added because resize handles grabbed otherwise, for feature 94. don't understand why?
```

Comment:
I still don't understand the need for the offsets here...

Reply:
Honestly, I don't understand it either.